tests/test_align.py

```diff
@@ -2,10 +2,14 @@
 from Bio.Seq import Seq
 from Bio.SeqRecord import SeqRecord
 
+from shlex import quote
+import pathlib
+
 from augur import align
 
 import pytest
 
+cur_dir = pathlib.Path(__file__+'/..').resolve()
```

Comment:
It should be safe to assume that the current working directory during test execution is the root of the project, so test fixtures can be referred to by simple strings like `"test/data/align/test_aligned_sequences.fasta"`.

Reply:
Ok.  I like that better.